### PR TITLE
added my tada68's from Gitlab.

### PR DESCRIPTION
**Pull request by sm0ggles**
Added two tada68 layouts.

### CODE REVIEW

File: keyboards/tada68/keymaps/sm0g/rules.mk [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+SLEEP_LED_ENABLE = yes       # Breathing sleep LED during USB suspend
+TAP_DANCE_ENABLE = yes      # Tappa Dance Bitch
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/tada68/keymaps/sm0g/rules.mk]: Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.

File: keyboards/tada68/keymaps/sm0gunix/config.h [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#include "../../config.h"
```
Comment (drashna) [inline keyboards/tada68/keymaps/sm0gunix/config.h]: This line is not actually needed, and should be removed. 

Specifically, the build script automatically includes all of the config.h files, so this is not necessary. 

And you should have a `#pragma once` at the top here.

File: keyboards/tada68/keymaps/sm0gunix/rules.mk [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+SLEEP_LED_ENABLE = yes      # Breathing sleep LED during USB suspend
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/tada68/keymaps/sm0gunix/rules.mk]: Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.

File: keyboards/tada68/keymaps/sm0g/keymap.c [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BL 0
+#define _FL 1
+#define _GM 2
+#define _GF 3
+
+#define _______ KC_TRNS
```
Comment (noroadsleft) [inline keyboards/tada68/keymaps/sm0g/keymap.c]: QMK defines this keycode alias automatically. This line can be deleted.

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L29-L31

File: keyboards/tada68/keymaps/sm0gunix/keymap.c [hunk 5]
```diff
@@ -0,0 +1,57 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BL 0
+#define _FL 1
+
+#define _______ KC_TRNS
```
Comment (noroadsleft) [inline keyboards/tada68/keymaps/sm0gunix/keymap.c]: QMK defines this keycode alias automatically. This line can be deleted.

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L29-L31

File: keyboards/tada68/keymaps/sm0g/rules.mk (hunk 1 above)
Comment (sm0ggles) [reply keyboards/tada68/keymaps/sm0g/rules.mk]: Oops. Yeah, completely forgot.

File: keyboards/tada68/keymaps/sm0g/keymap.c (hunk 4 above)
Comment (sm0ggles) [reply keyboards/tada68/keymaps/sm0g/keymap.c]: Removed. I actually forgot I even had those there. Lol.

**Comment by drashna** [comment]
Awesome, looks good! Thanks!

**Pull request merged**